src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/summary.html.twig

```diff
@@ -115,6 +104,29 @@
               <button class="btn btn-primary" type="submit" id="create-order-button">
                 {{ 'Create the order'|trans({}, 'Admin.Orderscustomers.Feature') }}
               </button>
+
+              <div class="btn-group">
+                <button
+                  class="btn btn-outline-primary dropdown-toggle"
+                  type="button"
+                  id="dropdown-menu-actions"
+                  data-toggle="dropdown"
+                  aria-haspopup="true"
+                  aria-expanded="false"
+                >
+                  {{ 'More actions'|trans({}, 'Admin.Orderscustomers.Feature') }}
+                </button>
+
+                <div class="dropdown-menu" aria-labelledby="dropdown-menu-actions">
+                  <button class="dropdown-item" id="js-send-process-order-email-btn">
+                    {{ 'Send pre-filled order to the customer by email'|trans({}, 'Admin.Orderscustomers.Feature') }}
+                  </button>
+                  <a class="dropdown-item" id="js-process-order-link">
+                    {{ 'Proceed to checkout in the front office'|trans({}, 'Admin.Orderscustomers.Feature') }}
+                  </a>
+                </div>
```

Comment:
```suggestion
                  {{ renderhook('displayAdminOrderCreateExtraButtons') }}
                </div>
```
Idea: what about adding this hook? this would allow modules to add more buttons in this dropdown

Reply:
Why not, nothing more needed than this ?

Reply:
@matks